### PR TITLE
Added Samples Table and Introduced configurable tables on ArmsDetails Page & Case Details Page. 

### PR DESCRIPTION
**Pull request by iksheth**
## Description
This PR covers solution for 4 tickets:  
BENTO-632: Associated Samples on Case Detail Page
BENTO-665: FE - Styling for Case Detail page
BENTO-673: Custodian configures table in Arm/Study Detail Page
BENTO-674: Custodian configures table in Case Detail Page

Fixes # (issue)
- Fixed Issue of Help message not popping up on help icon hover. 

Please delete options that are not relevant.
- [x] New feature (non-breaking change which adds functionality)

## How Has This Been Tested?
Yes on Local

### CODE REVIEW

File: src/pages/caseDetail/caseDetailView.js [hunk 1]
```diff
@@ -213,20 +214,51 @@ const CaseDetail = ({ data, classes }) => {
...
+                  </Grid>
+                </Grid>
+              </div>
+            </div>
+          ) : ''
+      }
+      {
+        filesTable.display
+          ? (
+            <div id="table_case_detail" className={classes.tableContainer}>
```
Comment (skiran86) [inline src/pages/caseDetail/caseDetailView.js:250]: Please update table id's as
- Files Table - table_case_detail_files
- Cases Table - table_case_detail_samples

**Review by skiran86 — CHANGES_REQUESTED** [review]

File: src/pages/caseDetail/caseDetailView.js (hunk 1 above)
Comment (iksheth) [reply src/pages/caseDetail/caseDetailView.js:250]: Sure.

**Review by AjayDoddapaneni — APPROVED** [review]
Looks Good!!! Once the above changes made good to go

**Comment by skiran86** [comment]
@iksheth Please take a look and update conflicting file

**Comment by iksheth** [comment]
working on it.

**Review by skiran86 — APPROVED** [review]

**Pull request merged**